### PR TITLE
Cuda 6.0.37

### PR DESCRIPTION
**Pull request by pescobar**
added %(version_major)s_%(version_minor)s to source_urls so it works with future major releases

reviewing the install_dir I see these folders which contains a few libraries and binaries:

```
libnvvp  nvvm  open64
```

but those are not included in LD_LIBRARY_PATH, LIBRARY_PATH and PATH in the generated modulefile. There is also a new folder "pkgconfig" which was not present in cuda-5.x releases.  

Maybe we should improve the cuda easyblock to add this in the modulefile? 

maybe @ajdecon can give some advice about this?

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/93afa79ed4cee3f913c6 for a full test report.

**Comment by boegel** [comment]
Can anyone else share the MD5sum for the 'source' file for this one?

Here's mine:

```
a36b96d42ccffd041ba45f2223ccb9b3  cuda_6.0.37_linux_64.run
```

File: easybuild/easyconfigs/c/CUDA/CUDA-6.0.37.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# http://developer.download.nvidia.com/compute/cuda/6_0/rel/installers/cuda_6.0.37_linux_64.run
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/c/CUDA/CUDA-6.0.37.eb]: hi, I'm the perpetrator of those comments and they are there for a reason:

occasionally, upstream providers shuffle their download locations; by keeping a copy of the download URL it makes much easier down the road to debug renaming of directories and restore easyconfig functionality...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c6c3d824a8e54a298bad for a full test report.

**Comment by boegel** [comment]
Turns out it was just a corrupt download on my end.

MD5 has been included to avoid situations like this in the future.

Thanks @pescobar, and everyone else for helping out with this one!

**Comment by fgeorgatos** [comment]
thanks for the work on this, guys

**Pull request merged**